docker-compose.yml

```diff
@@ -4,27 +4,41 @@ services:
     container_name: db
     build: ./db
     restart: always
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U sn -d stackernews"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
```

Comment:
:eyes: nice!

But I noticed that `-U` and `-d` don't do anything useful. It still returns the same even with bad values:

```
$ pg_isready
/run/postgresql:5432 - no response
$ pg_isready -p 5431
/run/postgresql:5431 - no response
$ pg_isready -h localhost -p 5431
localhost:5431 - accepting connections
$ docker exec -it db /bin/bash
root@d9b9bbf50f7b:/# pg_isready
/var/run/postgresql:5432 - accepting connections
root@d9b9bbf50f7b:/# pg_isready -h localhost
localhost:5432 - accepting connections
root@d9b9bbf50f7b:/# pg_isready -U foo
/var/run/postgresql:5432 - accepting connections
root@d9b9bbf50f7b:/# pg_isready -U foo -d bar
/var/run/postgresql:5432 - accepting connections
```

Reply:
I fixed this by querying postgres for tables instead